src/main/java/org/betonquest/betonquest/events/DropItemEvent.java

```diff
@@ -0,0 +1,166 @@
+package org.betonquest.betonquest.events;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.betonquest.betonquest.BetonQuest;
+import org.betonquest.betonquest.Instruction;
+import org.betonquest.betonquest.VariableNumber;
+import org.betonquest.betonquest.api.QuestEvent;
+import org.betonquest.betonquest.compatibility.protocollib.hider.EntityHider;
+import org.betonquest.betonquest.exceptions.InstructionParseException;
+import org.betonquest.betonquest.exceptions.QuestRuntimeException;
+import org.betonquest.betonquest.item.QuestItem;
+import org.betonquest.betonquest.utils.PlayerConverter;
+import org.betonquest.betonquest.utils.location.CompoundLocation;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityDamageEvent;
+import org.bukkit.event.entity.EntityPickupItemEvent;
+import org.bukkit.event.entity.ItemDespawnEvent;
+import org.bukkit.event.entity.ItemMergeEvent;
+import org.bukkit.event.player.PlayerJoinEvent;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+@SuppressWarnings({"PMD.CommentRequired", "PMD.FieldNamingConventions", "PMD.UnusedNullCheckInEquals", "PMD.CollapsibleIfStatements"})
```

Comment:
You should fix FieldNamingConventions, UnusedNullCheckInEquals and CollapsibleIfStatements instead off suppressing it